.pre-commit-config.yaml

```diff
@@ -4,7 +4,7 @@ repos:
     rev: 22.3.0
     hooks:
       - id: black
-        language_version: python3.7
```

Comment:
Python 3.7 is the oldest supported Python version and is still supported by Django 3.2. So, I don't think it is necessary for now to update it.

Reply:
I think this `language_version` here determines the python version to run the black on it.
As it was `py38` on tox, I've changed it to `python3.7`